go/stats/prombackend/prombackend.go

```diff
@@ -0,0 +1,63 @@
+package prombackend
+
+import (
+	"net/http"
+
+	prom "github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"vitess.io/vitess/go/stats"
+)
+
+// PromBackend implements PullBackend using Prometheus as the backing metrics storage
+type PromBackend struct {
+	namespace string
+}
+
+func InitPromBackend(namespace string) {
```

Comment:
Better golang style would be to just call this function `Init`.

You also need a doc comment of the form "// Init initializes the backend with the given namespace" to appease the linter.

More to the point you should install govet / golint / etc and hook it up with your editor.

Reply:
ooh TIL.

Reply:
Ran govet & golint on all the packages I added. Is there a convention for what to do for files in a package that you edited that you didn't modify that fail? (stats/ring.go fails)